Sources/Common/Communication/EventBusHandler.swift

```diff
@@ -144,7 +144,6 @@ public class CioEventBusHandler: EventBusHandler {
 
     /// Resets the EventBus to initial state by removing all observers and stored events.
     public func reset() async {
-        await eventStorage.removeAll()
```

Comment:
Since this is public api, I suggest we keep the name clear and not misleading. So may be rename this to `removeAllObservers` instead? 🤔